storagenode/storagenodedb/versions.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+// versions represents the database that contains the database schema version history.
+type versions struct {
+	location string
+	SQLDB
+}
+
+func newVersions(db SQLDB, location string) *versions {
+	return &versions{
+		location: location,
+		SQLDB:    db,
+	}
+}
+
+// Rebind rebind parameters
+func (db *versions) Rebind(s string) string { return s }
+
+// Schema returns schema
+func (db *versions) Schema() string { return "" }
```

Comment:
I agree with isaac, I think there should be a comment explaining why `Rebind` and `Schema` are currently noop and explaining the future plan

Reply:
Done!